pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -194,7 +194,8 @@
         "2.2.1",
         "2.2.2",
         "3.0.0",
-        "3.1.1"
+        "3.1.1",
+        "3.1.2"
```

Comment:
Do we want to produce a stable package in the next build? Why are we changing this?

Reply:
yes we want a new stable version of the package in july/next release.

Reply:
But that should be driven by the property that we set to produce stable packages which will update the index.json file automatically as part of the build, right?

Reply:
no we dont have that for 3.1, i am the one that generally does it. Sometimes it gets missed and then done after the release

Reply:
This is consistent with how we've been doing it in 3.1: https://github.com/Anipik/corefx/commit/bbcc0a83f548afb76bbde6226bb4a5ae06a980ab#diff-cb73a137838f73872d476fa5146bfede

Reply:
I think you're miss-understanding me. We update it once we've shipped the stable packages and prepare for the next release, but in order to produce stable packages we run this target:
https://github.com/dotnet/corefx/blob/344805d8ba3b42a1d6fc17b4a44a40a31b47d5b0/src/packages.builds#L50-L52

When DotNetFinalVersionKind is release. That target will get the packages we're building, will update the packageIndex.json in the build machine before building the packages and will produce stable packages.

Then, once we ship those to NuGet, you go and check-in those changes.

However, doing it here seems wrong, cause if we do a build were `DotNetFinalVersionKind != release` we will produce non-stable packages for all built packages of this release, and stable package for the platforms package. 

Do you see what I'm saying?

Reply:
> This is consistent with how we've been doing it in 3.1

Hmm then how do we produce non-stable servicing releases? That is the point of the target I pointed above.

Reply:
we no longer produces non-stable servicing release

Reply:
Ah ok, that is what was confusing me.

Reply:
I think we should be good either way